ReSharper.FSharp/src/FSharp/FSharp.Psi/src/Resolve/FSharpSymbolReference.cs

```diff
@@ -159,7 +159,24 @@ public FSharpOption<FSharpSymbolUse> ResolveWithFcs([NotNull] string opName, boo
         ? qualifiableReferenceOwner.Names
         : new[] {GetName()};
 
-      return checkerService.ResolveNameAtLocation(referenceOwner.FSharpIdentifier, names, resolveExpr, opName);
+      var symbolUses =
+        checkerService.ResolveNameAtLocation(referenceOwner.FSharpIdentifier, names, resolveExpr, opName);
+
+      return FilterSymbols(symbolUses);
     }
+
+    protected virtual FSharpList<FSharpSymbolUse> FilterSymbols(FSharpList<FSharpSymbolUse> symbols) =>
+      symbols.GetSlice(null, 0);
```

Comment:
What kind of filtering does this function do?

Reply:
Thank you, I can see the basic idea in the code. It is, however, unrelated to the question about the filtering.